Removed Python 2.7 Docker container and updated documentation (#5434)
cfpb/cfgov-refresh: docs/running-docker.md

We use [`docker-compose`](https://docs.docker.com/compose/reference/overview/)
to run an Elasticsearch container, a PostgreSQL container, 

> and Django in a Python 3.6 container. 

Reviewer: ```suggestion
and Django in a Python container. 
```

The following URLs are mapped to your host from the containers:

> - Access cfgov-refresh running in the Python 3.6 container: http://localhost:8000/

Reviewer: ```suggestion
- Access cfgov-refresh running in the Python container: http://localhost:8000/
```

### Environment variables

Environment variables from your `.env` file are sourced 
when the Python container starts

> and when you [access a running Python container](#access-a-container-shell). 

Reviewer: Similar comment about using "the" instead of "a" here, and the link target must retain the `s` in `containers` to match the slugified ID of the "Access a container's shell" heading.

```suggestion
and when you [access the running Python container](#access-a-containers-shell). 
```

Author: Done

> and when you [access the running Python container](#access-the-containers-shell). 

Reviewer: ```suggestion
and when you [access the running Python container](#access-a-containers-shell). 
```

### Access a container's shell

> - Python 3.6: `docker-compose exec python3 bash`

Reviewer: ```suggestion
- Python: `docker-compose exec python3 bash`
```

- Elasticsearch: `docker-compose exec elasticsearch bash`
- PostgreSQL: `docker-compose exec postgres bash`

### Run Django management commands

Django `manage.py` commands can only be run after you've 

> [opened up a shell in the Python container](](#access-the-containers-shell)). 

Reviewer: ```suggestion
[opened up a shell in the Python container](#access-a-containers-shell).
```

> [opened up a shell in a Python container](#access-a-container-shell). 

Reviewer: ```suggestion
[opened up a shell in the Python container](#access-a-containers-shell). 
```

and need to interact with the running Django process when the breakpoint is reached 
you can run [`docker attach`](https://docs.docker.com/engine/reference/commandline/attach/):

> - Python 3.6: `docker attach cfgov-refresh_python3_1`

Reviewer: ```suggestion
- Python: `docker attach cfgov-refresh_python3_1`
```

and need to interact with the running Django process when the breakpoint is reached 
you can run [`docker attach`](https://docs.docker.com/engine/reference/commandline/attach/):

> - `docker attach cfgov-refresh_python`

Reviewer: I believe the `_1` will still be needed in the formal container name used here.

```suggestion
`docker attach cfgov-refresh_python_1`
```

    docker-compose -f docker-compose.yml -f docker-compose.prod.yml up --build
    ```

>     This creates a container running Python 3.6 version of cf.gov, as well as

Reviewer: ```suggestion
    This creates a container running cf.gov on Python 3, as well as
```

1. Browse to your new local cf.gov site.

>     - http://localhost:8000

Reviewer: This is no longer a list.
```suggestion
   http://localhost:8000
```

>     - Python 3.6: http://localhost:8000

Reviewer: ```suggestion
   http://localhost:8000
```
